admin/twisted.spec

```diff
@@ -14,12 +14,13 @@ Group:		System/Libraries
 URL:		http://www.twistedmatrix.com/
 Requires:	python >= %{py_libver}
 BuildRequires:	python-devel
+BuildRequires:	pyproject-rpm-macros >= 1.2.0
```

Comment:
can we just remove this file here.
Is this file really use?
Do we have tests to make sure the files works?

I think is best to have this file stored in the fedora / rocky ...etc repos.

Reply:
Sure, removed!